cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICartoDBSink.java

```diff
@@ -618,18 +619,18 @@ protected String buildTableName(String servicePath, String entity, String attrib
                             + "dm-by-service-path data model");
                 } // if
 
-                name = NGSIUtils.encodePostgreSQL(servicePath, true);
+                name = NGSICharsets.cartoDBEncode(servicePath.substring(1));
```

Comment:
Good point. I already was thinking before pull requesting aboit it, but I did not obtain a conclussion... Removing it aligns this sink with all the other sinks, that remove the slash... well, except for STH :) It is true that our fresh new encoding mechanims can deal with it, and all the table names would start with `x002fx0000` (slash + concatenator), what could be a good "prefix" when looking for Cygnus tables in the DB xD